server/routes/product.js

```diff
@@ -43,7 +43,7 @@ router.get('/:productId', async (req, res) => {
  */
 router.post('/', async (req, res) => {
     const product = new Product({
-        title: req.body.title,
+        name: req.body.name,
```

Comment:
Yeah the name field is better. Have you tested it?

Reply:
Yeah!